src/ess/diffraction/powder/correction.py

```diff
@@ -44,3 +46,67 @@ def merge_calibration(*, into: sc.DataArray, calibration: sc.Dataset) -> sc.Data
         )
     out.masks['calibration'] = calibration['mask'].data
     return out
+
+
+def lorentz_correction(da: sc.DataArray) -> sc.DataArray:
```

Comment:
```suggestion
def apply_lorentz_correction(da: sc.DataArray) -> sc.DataArray:
```
or something like that, given that we changed what this function does?